returnn/tf/util/basic.py

```diff
@@ -9,6 +9,7 @@
 from tensorflow.python.client import device_lib
 from tensorflow.python.ops import init_ops
 import contextlib
+import numpy as np
```

Comment:
Do not import Numpy like that. Keep consistent to our other code. Use just `import numpy`.

Reply:
Done. Btw, is there a reason that we do it like this? It's quite unusual, isn't it?

Reply:
I think I have seen both quite often, so i would not say this is unusual. And I think I have seen `import numpy as np` mostly in quite hacky code or scripts only.

Reply:
Even on their own website, I always see `import numpy as np`. Also here
https://numpy.org/doc/stable/user/absolute_beginners.html#how-to-import-numpy

But doesn't really matter, I was just curious.